packages/oas/src/types.ts

```diff
@@ -112,7 +130,9 @@ export type OperationObject = (OpenAPIV3_1.OperationObject | OpenAPIV3.Operation
  * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#parameterObject}
  * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#parameterObject}
  */
-export type ParameterObject = OpenAPIV3_1.ParameterObject | OpenAPIV3.ParameterObject;
+export type ParameterObject = {
+  in: 'cookie' | 'header' | 'path' | 'query';
```

Comment:
Do the param objects we're using here not have `in` already/

Reply:
sort of, but it's typed as a string 😔

![CleanShot 2024-04-29 at 11 23 45@2x](https://github.com/readmeio/oas/assets/8854718/cb0efd19-ad99-405a-8ab2-26e4e52cbfb4)

Reply:
oh BROTHER